client/state/google-apps-users/test/reducer.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import assert from 'assert';
+
+/**
+ * Internal dependencies
+ */
+import {
+	GOOGLE_APPS_USERS_FETCH,
+	GOOGLE_APPS_USERS_FETCH_COMPLETED,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+
+import {
+	items,
+	loaded
+} from '../reducer';
+
+describe( 'reducer', () => {
+	describe( '#items()', () => {
+		it( 'should default to an empty array', () => {
+			const state = items( undefined, {} );
+
+			assert.deepEqual( state, [] );
+		} );
+
+		it( 'should return empty array for serialize', () => {
+			const state = items( [ 1, 2 ], { type: SERIALIZE } );
+
+			assert.deepEqual( state, [] );
+		} );
+
+		it( 'should return empty array for deserialize', () => {
+			const state = items( [ 1, 2 ], { type: DESERIALIZE } );
+
+			assert.deepEqual( state, [] );
+		} );
+
+		it( 'should return new items received', () => {
+			const state = items( [ { email: 'hello@world.com' } ],
+				{ type: GOOGLE_APPS_USERS_FETCH_COMPLETED, items: [ { email: 'hi@world.com' } ] } );
+
+			assert.deepEqual( state, [ { email: 'hello@world.com' }, { email: 'hi@world.com' } ] );
+		} );
+
+		it( 'should not have duplicate items', () => {
+			const state = items( [ { email: 'hello@world.com' } ],
```

Comment:
With the reducer tests, we also like to use deepFreeze to make sure we don't accidentally mutate current state. Usage looks a bit like:
```javascript
import deepFreeze from 'deep-freeze';
const originalState = deepFreeze( [ { email: 'hello@world.com' } ] );
const state = items( originalState, ...
```